requirements.txt

```diff
@@ -1,3 +1,4 @@
 mako>=1.0.2
 scrapy>=1.0.5
 urlobject
+lxml
```

Comment:
Pessimistic requirement seems reasonable to me. I'll do that, both for `lxml` and `urlobject`.